.github/workflows/main.yml

```diff
@@ -192,14 +192,12 @@ jobs:
         if: hashFiles('.coverage') != ''  # Rudimentary `file.exists()`
         run: |
           python -m pip install coverage
-          python -m coverage xml --ignore-errors
-        shell: C:\cygwin\bin\env.exe CYGWIN_NOWINPATH=1 CHERE_INVOKING=1 C:\cygwin\bin\bash.exe -leo pipefail -o igncr {0}
+          python -m coverage xml
+        shell: bash
```

Comment:
```suggestion
```

Is specifying the shell still needed here if you don't have any special argument ?

Reply:
Not sure.. To be explored.

Reply:
I find that `-o igncr` interesting in the original invocation through the Cygwin bash, though. It supposed to drop any carriage returns found.

My understanding is that on Windows, Git uses `\r\n` when checking out files from index into the working directory. And later on, that file is processed with only LF in mind which leaves CR as a part of the parsed YAML content which is how it ends up in the string that is later one written into a temporary script and passed to Bash which gets confused due to not treating CR as something special (rightfully so!) like Windows would.

Reply:
In the past, I used to enforce LF on all the text files in Git through `.gitattributes` but I know Jason doesn't like extra configs and tries relying on the default behavior of Git in hopes it'll do the right thing...

Reply:
> Is specifying the shell still needed here if you don't have any special argument ?

It's Powershell by default on Windows. It doesn't work the same way and can't even locate Python.

Reply:
FWIW I have several ideas to try out before I ask for help.